knowledge_repo/app/templates/markdown-base.html

```diff
@@ -89,6 +94,10 @@
   $(".btn-raw").on("click", function(){
     document.location.href = "/raw?markdown=" + encodeURI(post_path);
   })
+
+  $(".btn-webeditor").on("click", function(){
+    document.location.href = "/posteditor?path=" + encodeURI(post_path);
```

Comment:
Yup, totally agree! I believe the reason that it's done this way was that it's bad practice to mix jinja templating code with JS (which I did above for some reason, and should also be fixed). We should be able to take this out and replace it with a url_for in another PR.